plugin.video.srgssr_ch_replay/addon.py

```diff
@@ -64,12 +68,35 @@ def search_tv_shows(channel):
         query = {"bu": channel, "q": searchString}
         _query_tv_shows(channel, path, query, "searchResultListShow")
         
+        xbmcplugin.addSortMethod(pluginhandle, 1)
+        xbmcplugin.endOfDirectory(pluginhandle)
+        xbmcplugin.endOfDirectory(handle=pluginhandle, succeeded=True)
+        
+        
+def list_all_tv_shows(channel):
+    for c in '#' + ascii_lowercase:
+        _query_list_tv_shows(channel, c)
+        
+    xbmcplugin.addSortMethod(pluginhandle, 1)
+    xbmcplugin.endOfDirectory(pluginhandle)
+    xbmcplugin.endOfDirectory(handle=pluginhandle, succeeded=True) 
+
+
+def list_tv_shows(channel, characterFilter):
+    _query_list_tv_shows(channel, characterFilter)
+        
+    xbmcplugin.addSortMethod(pluginhandle, 1)
+    xbmcplugin.endOfDirectory(pluginhandle)
+    xbmcplugin.endOfDirectory(handle=pluginhandle, succeeded=True) 
+
 
-def list_tv_shows(channel, letter):
+def _query_list_tv_shows(channel, characterFilter):
     path = "/videometadata/v2/tv_shows/alphabetical"
-    query = {"bu": channel, "characterFilter": letter, "pageSize": "unlimited"}
+    query = {"bu": channel, "characterFilter": characterFilter, "pageSize": "unlimited" }
+    if not onlyActiveShows:
+        query.update({"onlyActiveShows": "false" })
```

Comment:
```suggestion
    query = {"bu": channel, "characterFilter": characterFilter, "pageSize": "unlimited", "onlyActiveShows": "true" if onlyActiveShows else "false"}
```

Reply:
nope, because it's an experimental feature and therefore this param should only be sent when the feature is enabled